.github/workflows/pr-lint-test-js.yml

```diff
@@ -13,12 +13,10 @@ jobs:
             - uses: actions/checkout@v3
 
             - name: Setup WooCommerce Monorepo
-              uses: ./.github/actions/setup-woocommerce-monorepo
-              with:
-                install-filters: 'woocommerce/client/admin...'
+              uses: ./.github/actions/setup-woocommerce-monorepo              
```

Comment:
The install filter was a headache here and causing failures for things that weren't installed. There may be a possible filter that'd work better in future but for now I've opted to install everything.